yaksh/forms.py

```diff
@@ -276,3 +276,27 @@ class CourseForm(forms.ModelForm):
     class Meta:
         model = Course
         fields = ['name', 'active', 'enrollment']
+
+class EditProfile(forms.Form):
```

Comment:
You can use model form here.
Check ProfileForm [here](https://github.com/FOSSEE/FOSSEE-Forum/blob/forum-v2/forums/forms.py)